src/wp-includes/class-wp-theme-json.php

```diff
@@ -1476,11 +1476,13 @@ protected static function get_style_nodes( $theme_json, $selectors = array() ) {
 		);
 
 		if ( isset( $theme_json['styles']['elements'] ) ) {
-			foreach ( $theme_json['styles']['elements'] as $element => $node ) {
-				$nodes[] = array(
-					'path'     => array( 'styles', 'elements', $element ),
-					'selector' => static::ELEMENTS[ $element ],
-				);
+			if ( array_key_exists( $element, static::ELEMENTS ) ) {
```

Comment:
hey @scruffian ,

This line containing the if check should be below the foreach check. 
Currently the test is failing because the $element is undefined.

Kindly check please.

Reply:
sorry about the typo.
I meant the `foreach loop`.

To be more clear: 
line 1480 should be at line 1479 
and
line 1479 should be at line 1480

Reply:
Sorry about that